code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -183,7 +183,7 @@ var/list/gaslist_cache = null
 		if(cached_gases["freon"])
 			if(cached_gases["freon"][MOLES] >= MOLES_PLASMA_VISIBLE)
 				if(holder.freon_gas_act())
-					cached_gases["freon"][MOLES] -= MOLES_PLASMA_VISIBLE
+					cached_gases["freon"][MOLES] -= 0.1

```

Comment:
Remove it entirely please I keep forgetting to now that scrubbers work on it properly